### PR TITLE
chore: update to electron 11

### PR DESCRIPTION
**Pull request by deepak1556**
Electron release link - https://github.com/electron/electron/releases/tag/v11.0.2

- [x] Keytar https://github.com/microsoft/vscode/pull/110815, https://github.com/microsoft/vscode/pull/110977
- [x] vscode-windows-ca-certs https://github.com/microsoft/vscode-windows-ca-certs/pull/1

### CODE REVIEW

File: build/azure-pipelines/product-build.yml
Comment (deepak1556) [inline build/azure-pipelines/product-build.yml:143]: For now we are cross-compiling but when ci supports native compilation, it will be using a different pool. Hence this is written as a separate stage.

File: package.json
Comment (deepak1556) [inline package.json:194]: The version bump cannot be done in current master because of new napi version.

File: build/azure-pipelines/product-build.yml
Comment (deepak1556) [inline build/azure-pipelines/product-build.yml:146]: Turning this off for now because internal mac arm builds are not available yet, there is active work to get a release out sometime next week.

File: package.json
Comment (deepak1556) [inline package.json:4]: This is just to fix conflicts with distro and get exploration builds out from this branch, will remove this before merging and fix conflicts on distro branch.

**Comment by deepak1556** [comment]
This is now ready for review, also last night's exploration release was made from this branch

File: .yarnrc
Comment (bpasero) [inline .yarnrc:2]: I think we are not yet having internal builds for this right?

**Review by bpasero — APPROVED** [review]

File: .yarnrc
Comment (deepak1556) [reply .yarnrc:2]: yup, will merge once the builds are available, can expect them in beginning of next week.

File: src/vs/code/test/electron-main/nativeHelpers.test.ts
Comment (chrmarti) [inline src/vs/code/test/electron-main/nativeHelpers.test.ts:43]: On a fresh Windows machine the cert store might have very few certificates because Edge downloads and adds to it on demand. Not sure if it can be empty, but if we see this test failing we need to consider that possibility.

**Review by chrmarti — APPROVED** [review]
Looks good. Thanks!

**Review by joaomoreno — APPROVED** [review]
lgtm

**Comment by bpasero** [comment]
@deepak1556 looks like we can also remove this code now, no?

https://github.com/microsoft/vscode/blob/38f55ece05a937a0821c4eeb15e9194a46b2d36e/src/vs/platform/windows/common/windows.ts#L129-L132

**Comment by deepak1556** [comment]
> @deepak1556 looks like we can also remove this code now, no?

Good catch, will remove it. Thanks!

**Comment by deepak1556** [comment]
@sbatten can you review https://github.com/microsoft/vscode/pull/110759/commits/c21f0012d26ae6fd50ca3674b1732fe0856b3156 which addresses https://github.com/microsoft/vscode/pull/110759#issuecomment-732082444

Reason for change: Electron has fixed the issue in https://github.com/electron/electron/pull/26395

**Review by sbatten — APPROVED** [review]

**Pull request merged**
